src/Amazon.CloudWatch.EMF/Model/MetricDirective.cs

```diff
@@ -80,6 +80,25 @@ internal List<List<string>> AllDimensionKeys
             }
         }
 
+        internal void PutDimension(DimensionSet dimensionSet)
+        {
+            // Duplicate dimensions sets are removed before being added to the end of the collection.
+            // This ensures the latest dimension key-value is used as a target member on the root EMF node.
+            // This operation is O(n^2), but acceptable given sets are capped at 10 dimensions
```

Comment:
can we change the comment to mention `30 dimensions`

Reply:
Fixed